app/controllers/new_admin/allocation_chart_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module NewAdmin
+  class AllocationChartController < ApplicationController
+    layout "new_admin"
+
+    def index
+      allocations = AllocationsAndUnalocatedUsersQuery.new(Allocation).call
+
+      @allocations = allocations.reduce([]) do |acc, allo|
+        allocation = { :name => allo.user.first_and_last_name,
+                       :project_name => allo.project ? allo.project.name : nil,
+                       :level => decorated_user(allo).level,
+                       :specialty => decorated_user(allo).specialty,
+                       :english_evaluation => decorated_user(allo).english_level,
+                       :skills => decorated_user(allo).skills,
+                       :allocation_end_at => build_date(allo.end_at),
+                       :user => allo.user }
```

Comment:
Why change the allocation to a hash?

Reply:
`@allocations` is missing some fields so I formatted it into a hash. Is there a better way of doing this?

Reply:
changed it